Geometry/EcalCommonData/plugins/DDEcalBarrelNewAlgo.cc

```diff
@@ -843,14 +1622,14 @@ void DDEcalBarrelNewAlgo::execute(DDCompactView& cpv) {
 
     const DDLogicalPart spmLog(spmName(), spmMat(), ((0 != spmCutShow()) ? ddspm : temp2));
 
-    const double dphi(360. * deg / (1. * spmNPerHalf()));
+    const double dphi(360._deg / (1. * spmNPerHalf()));
```

Comment:
What's the purpose of multiplication by 1?

Reply:
@cvuosalo - please, do not ask. This code is not maintained and I do not feel like modifying it. There is a lot of very old stuff. I'll make an attempt to modernise it for the dd4hep port.

Reply:
It would be helpful to explain in the PR description what changes are being made and which are not. If this code will be eliminated after the DD4hep migration, why even bother with the CLHEP change?

Reply:
I have mentioned it here https://github.com/cms-sw/cmssw/pull/27383 and here https://github.com/cms-sw/cmssw/pull/27395. There are lots of differences in the reco relval validation due to the units migration. I want to eliminate this factor in comparison between DD and DD4hep.